database/seeders/RoleSeeder.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\Role;
+use Illuminate\Database\Seeder;
+
+class RoleSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     */
+    public function run(): void
+    {
+        $roles = [
+            [
+                'name'  => 'super_admin',
```

Comment:
`RoleName::SUPER_ADMIN->value`

Reply:
Same goes for all of the roles and the permissions.

Reply:
Also, add description for each role and permission. We can change that later if needed.